----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:28:43,027

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 598

Watching for file changes with StatReloader
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:30:27,992

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 222

/root/awstest/awstest/awstest/settings.py changed, reloading.
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:30:28,308

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 598

Watching for file changes with StatReloader
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:30:37,913

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET / HTTP/1.1" 200 1818
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:30:44,528

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /container/ HTTP/1.1" 200 61
----------------------------------------------------------------------------------------------------
[ERROR] - 2020-06-13 04:30:44,529

Source log file: /root/awstest/awstest/ec2/ec2.py line: 37

Traceback (most recent call last):
  File "/root/awstest/awstest/ec2/ec2.py", line 33, in query
    self.response[target] = json.loads(body)
  File "/usr/lib64/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:30:44,530

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /ec2/ HTTP/1.1" 200 456
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:36:35,837

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 598

Watching for file changes with StatReloader
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:36:42,488

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET / HTTP/1.1" 200 1818
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:36:47,925

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /container/ HTTP/1.1" 200 61
----------------------------------------------------------------------------------------------------
[ERROR] - 2020-06-13 04:36:47,926

Source log file: /root/awstest/awstest/ec2/ec2.py line: 37

Traceback (most recent call last):
  File "/root/awstest/awstest/ec2/ec2.py", line 33, in query
    self.response[target] = json.loads(body)
  File "/usr/lib64/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:36:47,927

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /ec2/ HTTP/1.1" 200 456
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 04:37:01,804

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 598

Watching for file changes with StatReloader
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:50:47,420

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 598

Watching for file changes with StatReloader
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:50:52,767

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET / HTTP/1.1" 200 1818
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:50:52,997

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /static/js/jquery-3.5.1.min.js HTTP/1.1" 200 89476
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:50:53,037

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /static/css/bootstrap.min.css HTTP/1.1" 200 155758
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:50:53,057

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /static/js/bootstrap.min.js HTTP/1.1" 200 58072
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:50:53,356

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /static/js/bootstrap.min.js.map HTTP/1.1" 200 190253
----------------------------------------------------------------------------------------------------
[WARNING] - 2020-06-13 07:50:53,471

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/log.py line: 228

Not Found: /favicon.ico
----------------------------------------------------------------------------------------------------
[WARNING] - 2020-06-13 07:50:53,472

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /favicon.ico HTTP/1.1" 404 2200
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:50:53,581

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /static/css/bootstrap.min.css.map HTTP/1.1" 200 625953
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:50:59,287

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /container/ HTTP/1.1" 200 61
----------------------------------------------------------------------------------------------------
[ERROR] - 2020-06-13 07:50:59,289

Source log file: /root/awstest/awstest/ec2/ec2.py line: 37

Traceback (most recent call last):
  File "/root/awstest/awstest/ec2/ec2.py", line 33, in query
    self.response[target] = json.loads(body)
  File "/usr/lib64/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:50:59,290

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /ec2/ HTTP/1.1" 200 456
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:52:53,482

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET / HTTP/1.1" 200 1862
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:52:56,116

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /container/ HTTP/1.1" 200 61
----------------------------------------------------------------------------------------------------
[ERROR] - 2020-06-13 07:52:56,116

Source log file: /root/awstest/awstest/ec2/ec2.py line: 37

Traceback (most recent call last):
  File "/root/awstest/awstest/ec2/ec2.py", line 33, in query
    self.response[target] = json.loads(body)
  File "/usr/lib64/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:52:56,117

Source log file: /usr/local/lib64/python3.6/site-packages/django/core/servers/basehttp.py line: 157

"GET /ec2/ HTTP/1.1" 200 456
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:54:11,823

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 222

/root/awstest/awstest/ec2/ec2.py changed, reloading.
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:54:12,114

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 598

Watching for file changes with StatReloader
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:58:21,918

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 222

/root/awstest/awstest/ec2/ec2.py changed, reloading.
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 07:58:22,213

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 598

Watching for file changes with StatReloader
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 08:00:19,467

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 222

/root/awstest/awstest/ec2/ec2.py changed, reloading.
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 08:00:19,759

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 598

Watching for file changes with StatReloader
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 08:00:34,437

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 222

/root/awstest/awstest/ec2/ec2.py changed, reloading.
----------------------------------------------------------------------------------------------------
[INFO] - 2020-06-13 08:00:34,733

Source log file: /usr/local/lib64/python3.6/site-packages/django/utils/autoreload.py line: 598

Watching for file changes with StatReloader
